daemon/algod/api/algod.oas3.yml

```diff
@@ -350,6 +350,28 @@
         },
         "description": "Encoded block object."
       },
+      "BlockTxidsResponse": {
+        "content": {
+          "application/json": {
+            "schema": {
+              "properties": {
+                "blockTxids": {
+                  "description": "Block txids.",
```

Comment:
> Elsewhere, we always say "transaction ID", rather than txid in descriptions.

Good point -- I'll change this

> I'd also advocate saying "including inner transactions" when describing the endpoint.

We weren't going to include inner Tx IDs here (only top level IDs) -- is that a commonly requested feature?

Reply:
Oh, that's actually totally fine with me! I saw "DecodePaysetFlat" and jumped to conclusions, I didn't read closely.